src/test/resources/features/checksum_verification.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Verify checksums
+
+	Background:
+		Given the internet is reachable
+		And an initialised environment
+
+	Scenario: Install a specific Candidate with a valid SHA-256 checksum
+		Given the system is bootstrapped
+		And the candidate "grails" version "1.3.9" is available for download with checksum "1f9234c8e622ec46d33883ea45b39ede768b92d478fe08f6952548247f7fbb65" using algorithm "SHA-256"
+		When I enter "sdk install grails 1.3.9"
+		Then I see "Done installing!"
+		And the candidate "grails" version "1.3.9" is installed
+		And the response headers file is created for candidate "grails" and version "1.3.9"
+		And the exit code is 0
+
+	Scenario: Install a specific Candidate with a valid SHA1 checksum
+		Given the system is bootstrapped
+		And the candidate "grails" version "1.3.9" is available for download with checksum "c68e386a6deec9fc4c1e18df21f92739ba2ab36e" using algorithm "SHA1"
+		When I enter "sdk install grails 1.3.9"
+		Then I see "Done installing!"
+		And the candidate "grails" version "1.3.9" is installed
+		And the response headers file is created for candidate "grails" and version "1.3.9"
+		And the exit code is 0
+
+	Scenario: Install a specific Candidate with a valid md5 checksum
+		Given the system is bootstrapped
+		And the candidate "grails" version "1.3.9" is available for download with checksum "1e87a7d982a2f41da96fdec289908552" using algorithm "MD5"
+		When I enter "sdk install grails 1.3.9"
+		Then I see "Done installing!"
+		And the candidate "grails" version "1.3.9" is installed
+		And the response headers file is created for candidate "grails" and version "1.3.9"
+		And the exit code is 0
+
+	Scenario: If an unknown algorithm is used it should not fail
```

Comment:
Is it worth also having an explicit scenario for no algorithm present? Sometimes the data will not be present in the DB, so we won't be propagating headers at all in the broker service.

Reply:
I think this is already covered in all other install features (i.e. these fixtures don't generate the header files)

Reply:
I'll merge it, but will then add that scenario myself. Not for the sake of testing it, but more for explicitly documenting the behaviour.